src/languages/en.ts

```diff
@@ -6147,7 +6147,7 @@ const translations = {
         searchResults: {
             emptyResults: {
                 title: 'Nothing to show',
-                subtitle: `Try adjusting your search criteria or creating something with the green ${CONST.CUSTOM_EMOJIS.GLOBAL_CREATE} button.`,
+                subtitle: `Try adjusting your search criteria or creating something with the + button.`,
```

Comment:
@dubielzyk-expensify to answer your question, yes! We are removing the icon and the word "green" from all translations, as the button is not green anymore.